src/DistributedLock.Tests/Infrastructure/Redis/RedisServer.cs

```diff
@@ -35,7 +37,13 @@ private RedisServer(int? port, bool allowAdmin)
                 .RedirectStandardErrorTo(Console.Error);
             ActiveServersByPort.Add(this.Port, this);
         }
-        this.Multiplexer = ConnectionMultiplexer.Connect($"localhost:{this.Port}{(allowAdmin ? ",allowAdmin=true" : string.Empty)}");
+        this.Multiplexer = ConnectionMultiplexer.Connect(new ConfigurationOptions
+        {
+            EndPoints = { { "localhost", this.Port } },
+            AllowAdmin = allowAdmin,
+            AbortOnConnectFail = abortOnConnectFail, // for backward compatibility with pre-2.5 versions
```

Comment:
Added to fix https://ci.appveyor.com/project/madelson/distributedlock/builds/48169811/job/jub04w8545uudgbs; let me know if there's a better fix.